tests/usr.bin/cc/t_ubsan_int_sub_overflow.sh

```diff
@@ -152,32 +148,33 @@ int_sub_overflow32_body(){
 #include <limits.h>
 #include <stdio.h>
 #include <stdlib.h>
-int main(int argc, char **argv) { int l = INT_MIN; l-= argc; printf("CHECK\n"); exit(0); }
+int main(int argc, char **argv) { int l = INT_MAX; l-= argc; printf("%d\n", l); exit(0); }
 EOF
 
-	cc -fsanitize=undefined -o df32 -m32 test.c
-	cc -fsanitize=undefined -o df64 test.c
-	file -b ./df32 > ./ftype32
-	file -b ./df64 > ./ftype64
+	cc -fsanitize=undefined -o md32 -m32 test.c
+	cc -fsanitize=undefined -o md64 test.c
+	file -b ./md32 > ./ftype32
+	file -b ./md64 > ./ftype64
 	if diff ./ftype32 ./ftype64 >/dev/null; then
 		atf_fail "Generated binz ain't no different"
 	fi
 	echo "32bit Binz on this platform are:"
 	cat ./ftype32
 	echo "64bit Binz are on the other hand:"
 	cat ./ftype64
-	atf_check -s not-exit:0 -o not-match:"CHECK\n" -e match:"signed integer overflow" ./test
+    atf_check -o not-match:"^[-]?\d-([.]\d-)" -e match:"signed integer overflow" ./md32 #match that the output int is not positive(:overflown)
 
 	# Another test with profile 32bit binaries, just to make sure everything has been thoroughly done
 	cat > test.c << EOF
 #include <limits.h>
 #include <stdio.h>
 #include <stdlib.h>
-int main(int argc, char **argv) { int l = INT_MIN; l-= argc; printf("CHECK\n"); exit(0); }
+/* Alternatively we could just do a return ternary of the style "return (int == result of subtraction)?0:-1;"*/ 
```

Comment:
Please drop `/* */`. We don't want to keep perfecting it with this direction, no gain.

Reply:
This is a comment intended for your eyes :)